backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -393,13 +429,16 @@ public void rowsToRdf(final ModelBuilder builder, Table table, final String rowI
       if (table.getMetadata().getTableType() == TableType.ONTOLOGIES) {
         builder.add(subject, RDF.TYPE, IRI_CODED_VALUE_DATATYPE);
         builder.add(subject, RDF.TYPE, OWL.CLASS);
+        builder.add(subject, RDF.TYPE, SKOS.CONCEPT);
         builder.add(subject, RDFS.SUBCLASSOF, tableIRI);
-
+        builder.add(subject, SKOS.IN_SCHEME, tableIRI);
         if (row.getString("name") != null) {
           builder.add(subject, RDFS.LABEL, Values.literal(row.getString("name")));
+          builder.add(subject, SKOS.PREF_LABEL, Values.literal(row.getString("name")));
```

Comment:
If we use skos:prefLabel and skos:altLabel we don't need to also add a triple with rdfs:label, since these skos terms are subProperties of rdfs:label.